src/Backend/Form/Type/FileType.php

```diff
@@ -134,10 +134,20 @@ public function configureOptions(OptionsResolver $resolver)
             [
                 'data_class' => AbstractFile::class,
                 'empty_data' => function () {
-                    $emptyData = new StdClass();
-                    $emptyData->file = null;
+                    return new class extends StdClass {
```

Comment:
this class is only used internally and never leaves this class as this class also transforms it.

I used StdClass at first, while it worked it gave typehint errors in IDE's

I think this is the perfect usecase for anonymous classes